docs/controls/wpf-winforms/WindowsXamlHost.md

```diff
@@ -164,150 +179,10 @@ private void CreateUWPControlsFirst()
 
 private void MyButton_Click(object sender, Windows.UI.Xaml.RoutedEventArgs e)
 {
-    MessageBox.Show("My Button Worked");
+    MessageBox.Show("This is a UWP button.");
 }
 ```
 
-## Add a custom UWP control

```

Comment:
Are we going to have a separate article for this scenario/example?  If so, it appears to be missing?

Reply:
This was moved up to line 26 above, and yes there is now a separate article covering this scenario. This is probably the most important scenario for this control, and it is also the most complicated (and unfortunately still in flux as the feature continues to mature).

Reply:
Got it!  Thanks, so we'll have a separate PR later with more updates for that article.

Reply:
The referenced article has already been updated (https://docs.microsoft.com/en-us/windows/apps/desktop/modernize/host-custom-control-with-xaml-islands). Updating this WindowsXamlHost article in the toolkit docs to point to it is the last piece of remaining work for this round of doc updates.